go-controller/pkg/config/config.go

```diff
@@ -51,6 +51,7 @@ var (
 	// Kubernetes holds Kubernetes-related parsed config file parameters and command-line overrides
 	Kubernetes = KubernetesConfig{
 		APIServer:          "http://localhost:8080",
+		RawClusterSubnets:  "10.128.0.0/14/23",
```

Comment:
So is this really "Kubernetes-related"?

`Kubeconfig`, `CACert`, `APIServer`, and `Token` configure how ovn-kubernetes talks to Kubernetes. `OVNConfigNamespace` points to where we should operate inside Kubernetes. `ServiceCIDR` is a local copy of a kube-controller config option, which we will eventually be getting out of some k8s configuration API object instead.

But `RawClusterSubnets`/`ClusterNetworks` is ovn-kubernetes configuration, not Kubernetes configuration. Kubernetes itself doesn't know about it, or need to know about it.

(Same question/argument applies to the CLI argument renaming.)